docs/getting-started/auth.adoc

```diff
@@ -1,15 +1,169 @@
 = AeroGear Services Auth SDK
 
-Mobile authentication SDK based on Keycloak
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+Provides authentication features like access control and two factor authentication through Keycloak.
 
 == Adding dependency
 
 Add dependency to your application module
 ----
 dependencies {
-    implementation group: 'org.aerogear.android', name: 'auth'
+    implementation 'org.aerogear:auth:5.0.0-SNAPSHOT'
```

Comment:
@secondsun Is this the correct syntax?

Reply:
@aidenkeating That format without bom file. With bom you ignoring versions part. 
Rest is ok.